src/main.rs

```diff
@@ -87,8 +97,8 @@ fn main() {
 
         if let Some(threads) = opts.threads {
             if threads < 1 || threads > 4 {
-                error!("thread must be between 1 and 4 (inclusive)");
-                std::process::exit(1);
+                error!("threads must be between 1 and 4 (inclusive)");
```

Comment:
I know we didnt do this before but could you add a period at the end of each of these log statement messages?

Reply:
Yeah, why not